docs/DeveloperGuide.md

```diff
@@ -381,28 +391,27 @@ Use case ends.
 5. User self-grades question as easy/medium/difficult.  
    Repeat step 2-5 until all powercards in deck have been exhausted.
 
-Use case ends.
+   Use case ends.
 
 **Extensions:**
 
-&nbsp;&nbsp;&nbsp;&nbsp;1a. There are no decks with the name requested by the user.  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Use case ends
-
-&nbsp;&nbsp;&nbsp;&nbsp;*a. User decides to end the review early.  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*a1. User requests to end the review session  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;*a2. Powercard ends the session and brings the user back to default area  
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Use case ends
+* 1a. There are no decks with the name requested by the user.  
+  Use case ends.
 
+* *a. User decides to end the review early.  
+  * *a1. User requests to end the review session.  
+  * *a2. Powercard ends the session and brings the user back to the area before the review started.  
+    Use case ends.
 
 ### Non-Functional Requirements
 
-1.  Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
-2.  Should be able to hold up to 1000 Powercards without a noticeable sluggishness in performance for typical usage.
+1. Should work on any _mainstream OS_ as long as it has Java `11` or above installed.
+2. Should be able to hold up to 1000 Powercards without a noticeable sluggishness in performance for typical usage.
 3. A Powercard should load when prompted without any noticeable lag
 4. Every command should provide a response within 2 seconds
 5. The Powercard program is not expected to determine the correctness of the user’s response
 6. A Powercard should be easily added/deleted in less than 3 commands after opening the program
```

Comment:
Ours is maximum 3 commands right. addDeck, selectDeck, add.

Reply:
yep!